Library/Formula/makeheaders.rb

```diff
@@ -0,0 +1,24 @@
+require "formula"
+
+class Makeheaders < Formula
+  homepage "http://www.hwaci.com/sw/mkhdr/"
+  url "https://github.com/steakknife/makeheaders/archive/1.4-2.tar.gz"
```

Comment:
Write me a ticket, officer, for making code available to more people and platforms.